host_core/lib/host_core/host.ex

```diff
@@ -423,11 +423,15 @@ defmodule HostCore.Host do
       cluster_issuers: cluster_issuers(),
       invocation_seed: cluster_seed(),
       # In case providers want to be aware of this for their own logging
-      structured_logging_enabled: structured_logging_enabled
+      structured_logging_enabled: to_bool(structured_logging_enabled)
     }
     |> Jason.encode!()
   end
 
+  defp to_bool(val) when is_binary(val), do: String.downcase(val) == "true"
+
+  defp to_bool(_), do: false
```

Comment:
I don't think this case is currently possible -- since the config value defaults to [false](https://github.com/wasmCloud/wasmcloud-otp/blob/main/host_core/lib/host_core/config_plan.ex#L55) -- but if we ever change the default to `true`, will `to_bool` still return `false`?

Reply:
Agreed with Connor here, if it's a boolean we can just default to the boolean

Reply:
Oh good catch. I'll add that case